src/scenes/StopPlaces/scenes/Editor/index.js

```diff
@@ -113,10 +123,27 @@ const FlexibleStopPlaceEditor = ({ match, history }) => {
     [flexibleStopPlace, onFieldChange]
   );
 
-  const polygonCoordinates =
-    flexibleStopPlace && flexibleStopPlace.flexibleArea
-      ? flexibleStopPlace.flexibleArea.polygon.coordinates
-      : [];
+  const handleDrawPolygonClick = useCallback(() => {
+    let coords = JSON.parse(coordinates);
+    for (let coord of coords) {
+      if (coord[0] < coord[1]) {
```

Comment:
Why is this condition necessary? If you know you get a certain order, and you know you want to switch the order?

Reply:
Yeah, this is a shortcut, and should be improved somehow (only works for certain areas, like Norway). Maybe by being able to specify the lat/long-order or by upload and validation of some supported formats (like geojson). 

We know the order the editor expects (Lat,Long), but the lat/long-order from other sources varies (geojson has (Long,Lat)).

I would like to delay the upload/validation until we see that the feature brings value in practice, but maybe a way of letting the user specify the order is a good idea?

Reply:
Given that the graphql endpoint expects GeoJSON, and we tend to use GeoJSON in other apis, maybe we should just standardize on that until there is a need for a converter? I mean, we are not building a geometry data converter after all :-) The fact that leaflet uses lat,lon in its api is kind of irrelevant.

My suggestion is therefore: Make it clear in the UI that the field expects GeoJSON format, convert it for the purpose of the editor, make sure the correct format is still sent to the api.